apps/web/src/views/settings.js

```diff
@@ -818,6 +820,17 @@ function Settings() {
                 tip="Read details of all usage data we collect."
               />
             </Button>
+            {isDesktop() && (
```

Comment:
Privacy mode is not available on Linux so we should check for that here as well.